src/js/services/api/index.js

```diff
@@ -246,6 +246,11 @@ export const fetchFilteredPRs = async (
 
     if (filter.properties.length > 0) {
         filter_.properties = filter.properties;
+    } else {
+      filter_.properties = [
+        "wip", "reviewing", "merging", "releasing",
+        "release_happened", "rejection_happened"
```

Comment:
Using the constant names is clearer that we're using events (which are date-frame bounded) instead of stages (which are not date-frame bounded)
```suggestion
        PR_EVENT.RELEASE, PR_EVENT.CLOSURE_REJECTION
```